### PR TITLE
Exchange rate API

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@jbuck R?

File: tests/http/static-files.test.js [hunk 1]
```diff
@@ -99,14 +99,6 @@ describe('static files', () => {
       type: types.html,
       cache: cache.short
     },
-    '/exchange-rates/rates-backup.json': {
```
Comment (jbuck) [inline tests/http/static-files.test.js]: I think you should leave these here until they're actually removed

**Comment by alicoding** [comment]
@jbuck done :)

**Comment by jbuck** [comment]
@alicoding r+ on travis going green after re-addding the test

**Pull request merged**